### PR TITLE
Task 65 -66: Time travel

### PR DESCRIPTION
**Pull request by nrjvarshney**
Adding 2 Tasks:

1. MCQ
2. Classification

### CODE REVIEW

**Comment by danyaljj** [comment]
@nrjvarshney thanks for the PR. Looks like the PR is failing due to a JSON formatting issue (see the "details").

**Comment by danyaljj** [comment]
Run this locally to see the error: 
```
> python src/test_all.py
```

**Comment by nrjvarshney** [comment]
I've resolved the JSON formatting issues

**Comment by danyaljj** [comment]
Thanks! These are great contributions! 

Here are some thoughts for the first two tasks (will review the rest later). 

"task066_timetravel_classification"
 - the following seems necessary/redundant (repeats the definition), at least, to me: 
```
"\n Emphasis & Caution: The given sentence is incorrect if it changes the subsequent storyline, so that at least one of the three subsequent sentences no longer work Or in other words, the incorrect sentence and the original subsequent sentences can no longer form a consistent story."
```


"task065_timetravel_classification":  
- Similar comment as for the previous task 
- In the task description say what are the expected outputs: "option 1" and "option 2". 
- Minor capitalization issue in this file: `In Option 1, The new sentence` -> lower case `The` in the middle of a sentence 
- Add quotes wherever you're referring to a mention from the story. For example, `In Option 2, changed the action held their guns up to started yelling` might be written as `In Option 2, changed the action "held their guns up" to "started yelling" ...`  
- The explanation of the examples, should not refer to other examples (e.g., `changed the action held their guns up to started yelling`). These are not ordered examples; one might select a random subset of the examples. 

- please update the readme file that contains the task names and descriptions.

**Comment by danyaljj** [comment]
Thanks! One more round of comments: 


- The following "positive" example read coherent ("yes") to me: 

```
        {
            "input": "Sentence 1: The cashier was counting the dollar bills at her desk. \n Sentence 3: Everyone panicked and started to scream. \n  Sentence 4: The men threatened the people to remain quiet. \n Sentence 5: The cashier handed them the cash so they would go away.  \n Given Sentence 2: Two men rushed into the store and started yelling.",
            "output": "No",
            "explanation": "With the given sentence, sentence 5: 'cashier handed them the cash' is no longer consistent. So, 'No' is the correct answer."
        }
```

- Typo: `entty `

- Please merge the branch with the master.

**Review by swarooprm — COMMENTED** [review]
Looks good, some comments:
1. If a field is empty, don't add the header. E.g. in this case the field "emphasis & caution is empty". Instead of adding "emphasis& caution: ", you can remove it altogether. Similarly, emphasis & caution in task065.
2. There are some typos/grammatical inconsistencies, e.g. "2nd sentence" should be "the 2nd sentence". Make sure to read and resolve grammatical errors.
3. There seem to be some annotation issues: e.g. output in the following instances are the same as input, there are also similar examples
{
            "input": "Sentence 1: Charlie was starving. \n Sentence 2: He went into a gas station and tried to steal some tuna. \n **Sentence 3: A cop in the store noticed and stopped him \n Sentence 4:  Luckily, the cop let him go without charging him \n Sentence 5:  He felt so nervous as he walked away from the cop** \n New Sentence 2: He went into a gas station and tried to steal some tuna and will never feel nervous.",
            "output": [
                "**Sentence 3: A cop in the store noticed and stopped him. \n Sentence 4: Luckily, the cop let him go without charging him. \n Sentence 5: He felt so nervous as he walked away from the cop.**"
            ]
        },

**Comment by swarooprm** [comment]
Please pull and merge with the main branch. That should trigger the new automatic tests.

File: tasks/README.md [hunk 1]
```diff
@@ -67,6 +67,10 @@ Name | Summary | Category
 `task060_ropes_question_generation`	| Constructing questions regarding relations in the given paragraph.	| Question Generation
 `task061_ropes_answer_generation`	| Answering questions regarding relations in the given paragraph.	| Answer Generation
 `task062_bigbench_repeat_copy_logic`	|  Generating text that follows simple logical operations such as "repeat", "before", "after" etc.	| Logic
+`task063_timetravel_text_modification`	| Modifying a sentence in the given story to make the story inconsistent. | Text Modification
+`task065_timetravel_classification`	| Choosing the option that makes a given short story consistent. | Classification
+`task066_timetravel_classification`	| Identifying if the given sentence is consistent with the given story. | Classification
```
Comment (danyaljj) [inline tasks/README.md]: Optional: Make the titles more informative? Something like this: 
- `task065_timetravel_consistent_sentence_classification `
- `task066_timetravel_binary_consistency_classification`

File: tasks/README.md [hunk 2]
```diff
@@ -67,6 +67,10 @@ Name | Summary | Category
...
+`task073_CommonsenseQA_answer_generation`	|  Answering questions based on commonsense knowledge.	| Answer Generation
```
Comment (danyaljj) [inline tasks/README.md]: is this a repeated line?

**Comment by danyaljj** [comment]
Just to be sure: looks like you have dropped the long text generation (which is fine). Ideally, update the description of the PR.

**Comment by danyaljj** [comment]
Reviewed the tasks once more. I like 65 and 66, but not sure about 63 (task063_timetravel_text_modification) which seems a bit underdefined ("change a key aspect of 2nd sentence so that the story continuation changes"). Either we should make it more well-defined or drop it.

**Review by swarooprm — APPROVED** [review]
Task 065 and 066 look good! Merging.

**Comment by danyaljj** [comment]
There is some feedback from crowd workers regarding your task(s), summarized here: 
https://github.com/allenai/natural-instructions-expansion/pull/276#issuecomment-942714679
Wondering if you can address the feedback?

**Pull request merged**